site/layouts/partials/series_link.html

```diff
@@ -0,0 +1,39 @@
+{{ if .Params.series }}
+    {{ $cur_page_num := 1 }}
+    {{ $total_page_num := 0 }}
+    {{ $prev_link := "" }}
+    {{ $prev_title := "" }}
+    {{ $next_link := "" }}
+    {{ $next_title := "" }}
+    {{ $first_link := "" }}
+
+    {{ range where .Site.RegularPages.ByDate "Params.series" .Params.series }}
+        {{ $total_page_num = add $total_page_num 1 }}
+        {{ if gt $.Date.Unix .Date.Unix }}
+            {{ $cur_page_num = add $cur_page_num 1 }}
+            {{ $prev_link = .Permalink }}
+            {{ $prev_title = .Title }}
+        {{ end }}
+    {{ end }}
+    {{ range where .Site.RegularPages.ByDate.Reverse "Params.series" .Params.series }}
+        {{ $first_link = .Permalink }}
+        {{ if lt $.Date.Unix .Date.Unix }}
+            {{ $next_link = .Permalink }}
+            {{ $next_title = .Title }}
+        {{ end }}
+    {{ end }}
+    {{ if or $next_link $prev_link }}
+        <p>Part {{ $cur_page_num }} of {{ $total_page_num }}
+            in the <b>{{ .Params.series }}</b> series.</p>
```

Comment:
Nit: Make space it a bit different
```suggestion
        <p>
        	Part {{ $cur_page_num }} of {{ $total_page_num }} in the <b>{{ .Params.series }}</b> series.
        </p>
```